packages/components/src/components/vertical-tab/vertical-tab-header-title.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const VerticalTabHeaderTitle = ({ header_title }) => (
+    <div className='vertical-tab__header-title'>
```

Comment:
Please prefix all classes in the Components package with `dc-` (with updates to their respective stylesheet).